site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -197,7 +213,7 @@ include::_tenant-generator-length-config-rules.adoc[]
 :generator_display_name!:
 +
 
-[field]#tenant.externalIdentifierConfiguration.registrationVerificationIdGenerator.type# [type]#[String]# [required]#Required# [since]#Available since 1.8.0#::
+[field]#tenant.externalIdentifierConfiguration.registrationVerificationIdGenerator.type# [type]#[String]# [optional]#Optional# [default]#randomAlphaNumeric# [since]#Available since 1.8.0#::
```

Comment:
Was this always optional, or did it become optional in a recent release? If the latter, we should note.